src/wp-includes/class-wp-object-cache.php

```diff
@@ -159,10 +159,6 @@ protected function _exists( $key, $group ) {
 	 * @return bool True on success, false if cache key and group already exist.
 	 */
 	public function add( $key, $data, $group = 'default', $expire = 0 ) {
-		if ( wp_suspend_cache_addition() ) {
-			return false;
-		}
-
```

Comment:
Why is this being removed?

Reply:
Bc "add" calls "set" internally and this check has been added there. So it's removed to avoid duplicate checks